### ISSUE TITLE
.deb minor annoyances

### ISSUE THREAD

**Issue by dougK44883** [jessedp/tablo-tools-electron#18]
It's awesome you provide "ready to run" files (so much of github is do-it-yourself, not user friendly)  I suspect there's some automation involved to get so many platforms generated, maybe not.

I use the .deb running Debian 11 Bullseye -  the menu icon ``tapblo.png`` is put into ``/usr/share/icons/hicolor/0x0``  It's the only thing there... and it's not found, on my system anyhow.  Not being a developer, I can't say exactly, generally I find a "jumping off point" ``/usr/share/icons/hicolor/scalable``  ( sometimes ``/usr/share/pixmaps`` ) but what do I know... there's not been an icon in the menu for several revisions.

The sub menu, Programming ...instead of Sound & Video ??  Sometimes I'd edit the .desktop file [``/usr/share/applications/tablo-tools.desktop``] and add AudioVideo;Video; to the Categories list  leaving Development in place, so it shows up in a more practical place... from my (aka user) perspective.

If I copy the png to the scalable folder, unless it ever gets changed, I'll be done with it.  I can make the .desktop name slightly different so it will never get overwritten during an upgrade.  

It may be contributed to electron -  electron-userland/electron-builder#3609
I've done some superficial searching "/usr/share/icons/hicolor/0x0"  snap, appimage and ``gtk-update-icon-cache`` not recognizing it.

maybe when you get around to polishing 0.9.9 you can look into it, there's not impact on function as long as they can find it in the menus

#### Steps to Reproduce (for bugs)

1.

2.

3.

4.

#### Your Environment

- App version -
- Last 4 of Tablo ServerID -
- Tablo Firmware version -
- Operating System and version -

**Comment by jessedp** [comment]
Yeah, you found it - electron-builder does __all__ the heavy lifting to package this thing up and make it run. Otherwise it basically/is just a web app and a funky version of Chrome. 

I believe that icon stopped working when I upgraded electron or electron-builder, as you noted, a few versions ago. I have tried  to get it working again, so will make sure I revisit that.

The categories - ha. I have a feeling that's going to be easily fixed (and I guess I'll try to use that info) and more than likely saw it in a first build, thought no one's going to use it, and left it. And that I haven't done more than seeing if the deb/rpm/etc installed correctly occasionally rather than regular usage. Thankfully people tell me :)
